server/konnectors/venteprivee.js

```diff
@@ -0,0 +1,238 @@
+/*
+// This konnector retrieves invoices from website http://fr.vente-privee.com
+// creation : 11/06/2016
+// creator : https://github.com/SomeAverageDev
+*/
+'use strict';
+
+const request = require('request').defaults({
+  jar: true,
+  rejectUnauthorized: false,
+  followAllRedirects: true,
+  headers: {
+    'Accept-Language': 'fr,fr-FR;q=0.8,en-US;q=0.5,en;q=0.3',
+    'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:47.0) Gecko/20100101 Firefox/47.0',
+  },
+});
+
+const moment = require('moment');
+const cheerio = require('cheerio');
+const baseKonnector = require('../lib/base_konnector');
+
+const filterExisting = require('../lib/filter_existing');
+const localization = require('../lib/localization_manager');
+const saveDataAndFile = require('../lib/save_data_and_file');
+const linkBankOperation = require('../lib/link_bank_operation');
+
+const log = require('printit')({
+  prefix: 'vente-privee',
+  date: true,
+});
+
+const fileOptions = {
+  vendor: 'Vente-Privee',
+  dateFormat: 'YYYYMMDD',
+};
+
+const Bill = require('../models/bill');
+const baseUrl = 'https://secure.fr.vente-privee.com';
+
+// Konnector
+const connector = module.exports = baseKonnector.createNew({
+  name: 'Vente-Privee.com',
+  slug: 'vente-privee-com',
+  description: 'konnector description vente_privee',
```

Comment:
It uses the cozy-slug library to build the slug then it uses the key `konnector description built_slug`.